tests/integration/test_dynamodb.py

```diff
@@ -1570,7 +1571,7 @@ def test_nosql_workbench_localhost_region(self, dynamodb_create_table, aws_clien
         table = aws_client.dynamodb.describe_table(TableName=table_name)
         assert table.get("Table")
         # describe table for "localhost" region
-        client = aws_stack.connect_to_service("dynamodb", region_name="localhost")
+        client = connect_to(region_name="localhost").dynamodb
```

Comment:
Also connect_externally_to please

Reply:
Good catch, in fact the `aws_client_factory` fixture is the correct thing to use here. Fixed in c18bdc3916650b2f39310b701a6a90c013e57b38